src/components/VCalendar/mixins/times.ts

```diff
@@ -0,0 +1,97 @@
+import Vue from 'vue'
+
+import {
+  VTimestamp,
+  validateTimestamp,
+  parseTimestamp,
+  updateFormatted
+} from '../util/timestamp'
+
+export interface TimesObject {
+  now: VTimestamp
+  today: VTimestamp
+}
+
+export let times: TimesObject = {
+  now: parseTimestamp('0000-00-00 00:00') as VTimestamp,
+  today: parseTimestamp('0000-00-00') as VTimestamp
+}
+
+export default Vue.extend({
+  name: 'times',
+
+  props: {
+    now: {
+      type: String,
+      validator: validateTimestamp
+    }
+  },
+
+  data: () => ({
+    times
+  }),
+
+  computed: {
+    parsedNow (): VTimestamp | null {
```

Comment:
you do the `VTimestamp | null` here, :)

Reply:
The user might not pass the `now` prop to the component, in that case we go with `now` as determined by `new Date()`